app/jobs/publisher_removal_job.rb

```diff
@@ -17,7 +17,8 @@ def perform(publisher_id:)
       publisher.update(last_sign_in_ip: nil)
     end
     publisher.channels.pluck(:id).each do |channel_id|
-      DeletePublisherChannelJob.perform_now(channel_id: channel_id)
+      is_deleted = DeletePublisherChannelJob.perform_now(channel_id: channel_id)
+      raise ActiveRecord::Rollback unless is_deleted
```

Comment:
This is outside of the transaction, so this won't work.

Reply:
Good catch, shouldn't matter too much if these channels are being contested or not

https://github.com/brave-intl/publishers/pull/1936/commits/c15b4952c61888c26e6d4500d5772db9cf5667e2